src/base/gui/widget.cpp

```diff
@@ -163,8 +163,8 @@ Geom::AffineTransform Widget::getSelfTransform() const
 Geom::AffineTransform Widget::getTransform() const
 {
 	Geom::AffineTransform res;
-	if (parent) res *= parent->getTransform();
-	res *= getSelfTransform();
+	if (parent) res = res * parent->getTransform();
```

Comment:
unnecessary `res *` because it is an identity matrix.

Reply:
ok